js/build.js

```diff
@@ -191,25 +197,23 @@ if (typeof jQuery !== 'undefined') {
     $(this).addClass('sending');
     holder.addClass('sending');
 
-    $message.val('');
```

Comment:
You can't click the button twice because I removed the pointer events with CSS when `sending`